_includes/header.html

```diff
@@ -1,12 +1,8 @@
 <div class="ui teal inverted menu">
-  {% if include.page == 'home' %}
-  <a href="/" class="active item">
-    {% else %}
-    <a href="/" class="item">
-      {% endif %}
-      <i class="home icon"></i>
-      Home
-    </a>
+  <a href="/" class="{% if include.page == 'home' %}active {% endif %}item">
```

Comment:
Is this even used?

Reply:
Good catch, I completely forgot we removed this :+1: